app/uk/gov/hmrc/gform/controllers/helpers/FormDataHelpers.scala

```diff
@@ -39,6 +40,9 @@ object FormDataHelpers {
 
   private val logger = LoggerFactory.getLogger(getClass)
 
+  /** This filters HTML Entities (decimal/hex) or unicode from request body when entered.
+    * The pdf generator does not accept these characters.
+    */
```

Comment:
This comment is not correct. Problem is when we try to render (as markdown) SmartString which contains expression which refers to user input. Example is `summaryDescription` json field.